### ISSUE TITLE
Problem in the Pre-trained model

### ISSUE THREAD

**Issue by Almadani92** [bardiadoosti/HOPE#19]
The provided pre-trained model is not accurate. I tested it on several samples of the HOnnotate dataset but the results totally deviated from what it should be. even the resultant testing loss was very high. 
I know that this model has been trained on the First-hand dataset, but it should also give a close estimation on other datasets.
In the first place, I thought that the problem was on pre-processing the data before feeding it into the network, but this was not the case since I managed to re-train the model on HOnnotate and I managed to get a better estimation on the same dataset used in the training. However, I am still facing the same problem when I test it on data that was not being used in the training.
I am afraid that the code is causing over-fitting.
Please advise.

**Comment by bardiadoosti** [comment]
Hi, the pre-trained model is for first-person hand action dataset and because the 2D to 3D convertor model is completely dependent to the camera's intrinsic and extrinsic parameters it will not work on another dataset with totally different condition. As mentioned in the paper the results on the HOnnotate dataset's test set are from a model trained with HOnnotate's training set.
